lex_dup2.py returned with 1

Traceback (most recent call last):
  File "/home/herczy/devel/ply/test/lex_dup2.py", line 31, in <module>
    lex.lex()
  File "/home/herczy/devel/ply/ply/lex.py", line 899, in lex
    raise SyntaxError("Can't build lexer")
SyntaxError: Can't build lexer

ERROR: /home/herczy/devel/ply/test/lex_dup2.py:22: Rule t_NUMBER redefined. Previously defined on line 18
